ADVANCED.md

```diff
@@ -1110,6 +1116,8 @@ return (
 | `searchQuery`               | [Query Config](#query-config)                                           | optional  | {}      | Configuration options for the main search query.                                                                                                                                                                             |
 | `trackUrlState`             | Boolean                                                                 | optional  | true    | By default, [Request State](#request-state) will be synced with the browser url. To turn this off, pass `false`.                                                                                                             |
 | `urlPushDebounceLength`     | Integer                                                                 | optional  | 500     | The amount of time in milliseconds to debounce/delay updating the browser url after the UI update. This, for example, prevents excessive history entries while a user is still typing in a live search box.                  |
+| `a11yNotifications`         | Boolean                                                                 | optional  | false   | Search UI will create a screen reader accessible live region to announce search results & other actions. This will be turned on by default in our 2.0 release.                                                               |
```

Comment:
See https://github.com/elastic/search-ui/pull/359/commits/6c1dc877e498f5f6b8112d321320a62b220e648c#diff-f959751a4f181dc8bdc2673b10144d0fR1118 for proposed documentation in 2.0 (once this is released with true by default).